src/server/twitterSearch.js

```diff
@@ -23,43 +33,81 @@ const makeTweeterFromRaw = (raw) => {
     .build();
 };
 
-export const searchAndSave = (res, query) => {
-  T.get('search/tweets', { 'q': query, 'count': 300 }, function (err, result, response) {
-    let count = 0;
-    result.statuses.forEach((tweetRaw) => {
-      const userRaw = tweetRaw.user;
+const processTweet = (tweetRaw, isRetweet = false) => {
+  const tweeter = makeTweeterFromRaw(tweetRaw.user);
+  const retweetedStatusRaw = tweetRaw.retweeted_status;

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'retweeted_status' is not in camel case.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2175507150/issues/source?bid=3148782&fileBranchId=3163167#l38)